src/components/Registration/StartDate.vue

```diff
@@ -13,9 +13,8 @@
           :nudgeRight="40"
           :nudgeTop="85"
           :initialValue="getRegistration.startDate"
-          :minDate="startDateMinStr"
-          :maxDate="startDateMaxStr"
-          :inputRules="getShowErrors ? startDateRules : []"
```

Comment:
These rules were redundant, since the date picker does not allow you to select a date less than minDate or greater than maxDate.

This Create UI code is now almost the same as in Edit UI.

Reply:
What if the start date is empty? Shouldn't we show "Business Start date is required"?

Reply:
There is an existing validation for this:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/06dba28c-55d3-46c9-89f5-841449b148b3)

Reply:
Oh OK, I missed that. Sounds good Sev!